applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -69,21 +69,13 @@ To set up a cloud service to work with the application firmware, complete the st
 * AVSystem's LwM2M Coiote Device Management - :ref:`server_setup_lwm2m`.
   No additional configuration is needed if the server is set up according to the linked documentation.
 
-nRF Asset Tracker project
-=========================
-
-The application firmware is a part of the `nRF Asset Tracker project`_, which is an open source end-to-end example that includes cloud backend and frontend code for the following cloud providers:
-
-* AWS IoT Core - `Getting started guide for nRF Asset Tracker for AWS`_.
-* Azure IoT Hub - `Getting started guide for nRF Asset Tracker for Azure`_.
-
-The `nRF Asset Tracker project`_ contains separate documentation that goes through setup of the firmware and cloud-side setup.
-If you want to use the nRF Asset Tracker, follow the `nRF Asset Tracker project`_ documentation instead of the Asset Tracker v2 documentation.
+You can also use the nRF Asset Tracker project, which is a single cloud solution supporting multiple cloud providers.
+See the :ref:`nRF_asset_tracker` section for more details.
 
 .. _atv2_lwm2m_carrier_support:
 
-Carrier library support
-***********************
+LwM2M carrier library support
```

Comment:
I'd prefer if we could leave `LwM2M` out as it might confuse customers now that we also support an LwM2M-based cloud for sensor data. But maybe that's more of a renaming of the carrier library thing. In general, I don't see a reason that the protocol should be in the carrier library name.
cc @nordic-hani

Reply:
Second that

Reply:
The LwM2M wording was originally removed to prevent users from being confused when wanting to configure the application for lwm2m.